rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { shallowMount } from "@vue/test-utils";
+import RundeckInfo from "./RundeckInfo.vue";
+
+jest.mock("../../../rundeckService", () => ({
+  getRundeckContext: jest.fn().mockReturnValue({ rdBase: "mocked-rdBase" }),
+  url: jest.fn().mockReturnValue("mocked-url"),
+}));
+
+const mountRundeckInfo = async (props?: Record<string, any>) => {
+  return shallowMount(RundeckInfo, {
```

Comment:
Was there an issue that prevented you from using mount? So far from what I remember, tests seemed to be running fine with it.